### ISSUE TITLE
stream_socket_client()

### ISSUE THREAD

**Issue by Ordovicium** [RainLoop/rainloop-webmail#1394]
**Expected behavior and actual behavior:**

I changed on my shared host provider to PHP 7 where I get now the message for every domain: stream_socket_client() has been disabled for security reasons

Is there a work around or setting to use a different methode?

**Comment by ESVDESIGNER** [comment]
In order to eliminate this error message you need to do ONE of the following things:

Remove the stream_socket_client string from the disable_functions at php.ini* file
Ask your hosting provider to remove the string above if you don't have an access to the php.ini* file
Change hosting provider which allows the running of the stream_socket_client function.

http://www.2by2host.com/articles/php-errors-faq/disabled_stream_socket_client/

**Comment by Ordovicium** [comment]
I don't have access to php.ini and I don't like to change my hosting provider. Therefore I asked for a work around or different methode. Is fsockopen equal?

**Comment by RainLoop** [comment]
No, there are no other possibilities. But if your provider blocks the stream_socket_client function, then most likely all such things will also be blocked.
